.github/workflows/conformance.yml

```diff
@@ -47,6 +47,7 @@ jobs:
         - bindings.mqtt-vernemq
         - bindings.redis
         - bindings.rabbitmq
+        - bindings.postgres
```

Comment:
Nit, would be great if this list were sorted!

Reply:
Yes. Doesn't have to be in this PR though.